c++/Main.cpp

```diff
@@ -0,0 +1,21 @@
+#include "Nbody.cpp"
+#include "getPos.cpp"
+#include "PythonRead.cpp"
+#include <chrono>
+int main() {
+    auto start_time = std::chrono::high_resolution_clock::now();
+    std::vector<Nbody> nbodies = createNbodiesFromFile("latest_checkpoint.txt");
```

Comment:
I made them all vectors because there is an external library I may use in the future that speeds up processing regarding vectors, I will test and let you know.

Reply:
Epic, was just saying, what is a bit more important is what i said below. But i see you've already got to that.